### PR TITLE
Adding minimal builders page and personal page + fancy header improvement

### PR DESCRIPTION
**Pull request by fels21**
## Description

- The minimum viable page for builders has been added, with a static array where each can add their address.
- My personal page related to #8 has been added.

## Additional Information

- The minimum viable page for builders has been added to the header menu
- Some icons have been added to the header.

## Related Issues
#8 

fels21.eth

### CODE REVIEW

File: packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                  <li>
+                    <Link href="https://x.com/I_Cherny" className="link">
+                      Twitter
+                    </Link>
+                  </li>
+                  <li>
+                    <Link href="https://github.com/fels21" className="link">
+                      GitHub
+                    </Link>
+                  </li>
```
Comment (carletex) [inline packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx:67]: NextJS `<Link>` is meant to be used for internal navigation (prefetching, etc) but not for external link. Let's use `<a>` with `target="_blank"`

File: packages/nextjs/app/builders/page.tsx
Comment (carletex) [inline packages/nextjs/app/builders/page.tsx:1]: Let's remove this file for now and tackle the whole thing while working on https://github.com/BuidlGuidl/batch7.buidlguidl.com/issues/3

File: packages/nextjs/components/Header.tsx [hunk 2]
```diff
@@ -18,6 +18,12 @@ export const menuLinks: HeaderMenuLink[] = [
   {
     label: "Home",
     href: "/",
+    icon: <HomeIcon className="h-4 w-4" />,
+  },
+  {
+    label: "Builders",
+    href: "/builders",
+    icon: <UsersIcon className="h-4 w-4" />,
```
Comment (carletex) [inline packages/nextjs/components/Header.tsx:26]: Same, let's leave this for https://github.com/BuidlGuidl/batch7.buidlguidl.com/issues/3

**Review by carletex — CHANGES_REQUESTED** [review]
Hey @fels21 thanks for this, good job!

The page is looking great (on mobile too!). I left some review comments. Let me know when you tackle them and I will review again.

File: packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+const BuilderPage: NextPage = () => {
+  return (
+    <>
+      <Link href="/builders" passHref>
+        <div className="flex items-center text-blue-500 hover:text-blue-700">
+          <ArrowLeftIcon className="h-5 w-5 mr-2" />
+          Back to Builders
+        </div>
+      </Link>
```
Comment (carletex) [inline packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx:13]: This doesn't exist now, but it's ok... it'll soon :P

**Review by carletex — APPROVED** [review]

File: packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx (hunk 3 above)
Comment (fels21) [reply packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx:13]: 🤦‍♂️🤦‍♂️🤦‍♂️

**Pull request merged**